src/caselawclient/models/documents.py

```diff
@@ -251,7 +253,8 @@ def get_manifestation_datetimes(
             f"/akn:FRBRdate{name_filter}/@date",
             {"akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0"},
         )
-        return [datetime.datetime.fromisoformat(event) for event in iso_datetimes]
+
+        return [parse_string_date(event, pytz.UTC) for event in iso_datetimes]
```

Comment:
Is this the right place to define the timezone -- can we guarantee other XML non-specified times are UTC?

I'm pretty sure all the other ones are specified, and would be UTC if they're not.

Reply:
I'm going to go with yes, because we're only sure about it in this context (and even then it's only a 90% certainty)